save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/ReactorUtils.kt

```diff
@@ -179,3 +263,12 @@ fun downloadFromClasspath(
                 logger.error("$resourceName is not found on the classpath; returning HTTP 404...")
                 lazyResponseBody()
             }
+
+/**
+ * @param seed the starting value of this sequence.
+ * @return the sequence infinitely returning [seed].
+ */
+fun <T : Any> infiniteSequenceOf(seed: T): Sequence<T> =
```

Comment:
why do we need this method? it replaces a single call of method with a single lambda to a single call of method with a a single parameter

Reply:
Addressed with 8922ec9b9fc1eb42e0e5fcd3c715a9b0aa461675.